documentation/source/overview/concepts/pam50.rst

```diff
@@ -53,33 +72,44 @@ The atlas folder also contains an ``info_label.txt`` file to explain what each f
 .. include:: info_label-atlas.txt
    :code:
 
+.. note:: For more details on the implementation of atlas, see the `White matter atlas script <https://github.com/spinalcordtoolbox/spinalcordtoolbox-dev/tree/master/dev/atlas>`_. 
+
 
 .. _pam50-spinal-levels-section:
 
-Spinal levels (``PAM50/spinal_levels``)
-=======================================
+Spinal levels
+=============
 
-.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/pam50/spinal_levels_illustration.png
-    :figwidth: 40%
+**Location:** ``PAM50/template/PAM50_spinal_levels.nii.gz``
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/pam50/spinal_levels.png
```

Comment:
I still find the pink writing a bit too large

Reply:
Fixed in https://github.com/spinalcordtoolbox/doc-figures/commit/5a9ca65d6c2a9407a47d20bd63189426bf553942